external/netstandard/netstandard.depproj

```diff
@@ -22,6 +22,10 @@
       <!-- only for netfx verticals will this project provide runtime assets -->
       <RuntimePath>$(RuntimePath)</RuntimePath>
     </BinPlaceConfiguration>
+    <BinPlaceConfiguration Include="$(Configuration)">
```

Comment:
Why do we need this duplicated configuration? Why wouldn't the one you add in dir.targets not pick this up?